website/docs/developer/research/research-localstack.md

```diff
@@ -59,6 +59,323 @@ Docker's localhost can be accessed on the parent machine at http://host.docker.i
     ```
     [Full example here.](https://git.cr.usgs.gov/LPDA/appeears/-/blob/feature/rds-setup/deployment/terraform/dev/main.tf)
 
+
+### Setup details
+
+-	Make sure you have Docker desktop installed in your laptop and approved by EROS helpdesk.
```

Comment:
I would put this in an important admonition. See the styling guide here https://nasa.github.io/cumulus-orca/docs/developer/development-guide/documentation/documentation-style-guide#admonitions

I would also change `EROS Helpdesk` to `your local IT and security` to make this a little broader as other developers from other DAACs or teams could work on ORCA not just EROS people.